flamedisx/xenon/x1t_sr1.py

```diff
@@ -96,18 +125,32 @@ def cal_bias_tf(sig, fmap, domain_def, pivot_pt):
     return bias_out
 
 
-##
-# Loading combined cuts acceptances
-##
-path_cut_accept_s1 = ['S1AcceptanceSR1_v7_Median.json']
-path_cut_accept_s2 = ['S2AcceptanceSR1_v7_Median.json']
+def cal_rec_efficiency_tf(sig, fmap, domain_def, pivot_pt):
+    """ Computes the reconstruction efficiency given the pivot point
+    :param sig: photon detected
+    :param fmap: map returned by read_maps_tf
+    :param domain_def: domain returned by read_maps_tf
+    :param pivot_pt: Pivot point value (scalar)
+    :return: Tensor of bias values (same shape as sig)
+    """
+    tmp = tf.convert_to_tensor(sig, dtype=fd.float_type())
 
+    bias_median = tfp.math.interp_regular_1d_grid(x=tmp,
+            x_ref_min=domain_def[0], x_ref_max=domain_def[1], y_ref=fmap[1],
+            fill_value='constant_extension')
```

Comment:
I'd split this off into a separate function since these lines are duplicated several times (lines 98, 113, 116, 138, 143, 148). Maybe something like:
```python
def tf_interpolated(x, y, domain):
    x_tf = tf.convert_to_tensor(x, dtype=fd.float_type())  # Not sure if this line is needed in every case though
    return tfp.math.interp_regular_1d_grid(x=x_tf,
            x_ref_min=domain[0], x_ref_max=domain[1], y_ref=y,
            fill_value='constant_extension')
```